src/App.js

```diff
@@ -2,12 +2,89 @@ import React, { useState } from 'react';
 import { createMuiTheme, CssBaseline } from '@material-ui/core';
 import { ThemeProvider } from '@material-ui/styles';
 import color from '@edma/design-tokens/js/color';
+import fontSize from '@edma/design-tokens/js/fontSize';
+import font from '@edma/design-tokens/js/font';
+import weight from '@edma/design-tokens/js/weight';
+import spacing from '@edma/design-tokens/js/spacing';
 import NavigationContainer from './Components/Scenes/Chrome/Navigation/Navigation-Container';
 
 export default function App() {
   // Define Kirby theme colors using EDMA Design Tokens in a hook,
   // so that we can let users customize them later.
   const [theme, setTheme] = useState({
+    typography: {
```

Comment:
This has actually been externalized into its own module; Theme.js. I think we might need to rebase this against `dev` branch to make sure it's up to date.